### DISCUSSION TITLE
Get color from text

### DISCUSSION THREAD

**Discussion by tronghieu60s** [jaredh159/tailwind-react-native-classnames#251] [Ideas]
i want to get color from tailwind, example, i want to create a function getColor('bg-blue-400') => 'rgb(96 165 250)' or '#60A5FA'.
I don't know if this function exists yet

**Comment by jaredh159** [comment]
there's already a function for this: `tw.color('bg-blue-400')`, see here:

https://github.com/jaredh159/tailwind-react-native-classnames#api
